samples/excel/89-preview-apis/chart-axis.yaml

```diff
@@ -0,0 +1,246 @@
+script:
+    content: "$(\"#setup\").click(() => tryCatch(setup));
+    $(\"#get-axis-unit\").click(() => tryCatch(getAxisUnit));
+    $(\"#change-axis-unit\").click(() => tryCatch(changeAxisUnit));
+    $(\"#remove-axis-label\").click(() => tryCatch(removeAxisLabel));
+    $(\"#show-axis-label\").click(() => tryCatch(showAxisLabel));
+    $(\"#set-axis-title\").click(() => tryCatch(setAxisTitle));
+    
+    async function getAxisUnit() {
+        await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getItem(\"Sample\");
+    
+            let chart = sheet.charts.getItemAt(0);
+            let categoryAxis = chart.axes.categoryAxis;
+            let valueAxis = chart.axes.valueAxis;
+    
+            categoryAxis.load();
+            valueAxis.load();\_
+           
+            await context.sync();
+    
+            OfficeHelpers.UI.notify(\"The vertical axis display unit is: \" +  valueAxis.displayUnit);
+        });
+    }
+    
+    async function changeAxisUnit() {
+        await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getItem(\"Sample\");
+    
+            let chart = sheet.charts.getItemAt(0);
+            let axis = chart.axes.valueAxis;
+            axis.displayUnit = \"Hundreds\";
+    
+            await context.sync();
+    
+            OfficeHelpers.UI.notify(\"The vertical axis display unit has been changed to : \" + axis.displayUnit);
+        });
+    }
+    
+    async function removeAxisLabel() {
+        await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getItem(\"Sample\");
+    
+            let chart = sheet.charts.getItemAt(0);
+            let axis = chart.axes.valueAxis;
+            axis.showDisplayUnitLabel = false;
+    
+            await context.sync();
+    
+            OfficeHelpers.UI.notify(\"The vertical axis display unit has been changed to : \" + axis.showDisplayUnitLabel);
+        });
+    }
+    
+    async function showAxisLabel() {
+        await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getItem(\"Sample\");
+    
+            let chart = sheet.charts.getItemAt(0);
+            let axis = chart.axes.valueAxis;
+            axis.showDisplayUnitLabel = true;
+            await context.sync();
+    
+            OfficeHelpers.UI.notify(\"The vertical axis display unit has been changed to : \" + + axis.showDisplayUnitLabel);
+        });
+    }
+    
+    async function setAxisTitle() {
+        await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getItem(\"Sample\");
+    
+            let chart = sheet.charts.getItemAt(0);
+            let categoryAxis = chart.axes.categoryAxis;
+            categoryAxis.title.text = \"Bicycle parts\";
+    
+            let valueAxis = chart.axes.valueAxis;
+            valueAxis.title.text = \"Number of items\";
+            valueAxis.hasTitle = true;
```

Comment:
Once you set the title text, I believe you no longer need to do "hasTitle".  Setting the title text automatically turns on visiblity (or at least, it does for all chart things of bygone like chart title)

Reply:
But speaking of... where is **hasTitle** coming from?  Is this a new API?  Could you please start a thread with me and the ecosystem team, for why we're doing `hasTitle` instead of `valueAxis.title.visible`?

Reply:
ok.

Reply:
yes  hasTitle is a new API.  Will start a thread with you and the ecosystem team.